delphi/translators/for2py/rectify.py

```diff
@@ -1085,6 +1088,7 @@ def handle_tag_type(
             elif (
                     child.tag == "component-decl"
                     or child.tag == "component-decl-list"
+                    or child.tag == "component-decl-list__begin"
```

Comment:
I think something like `child.tag in ["component-decl", "component-decl-list", "component-decl-list__begin"]` might be simpler?